docs/source/pypureclient.flasharray.rst

```diff
@@ -18,6 +18,7 @@ Subpackages
    pypureclient.flasharray.FA_2_17
    pypureclient.flasharray.FA_2_19
    pypureclient.flasharray.FA_2_2
+   pypureclient.flasharray.FA_2_20
```

Comment:
nit: I assume we'll need to resolve sorting here

Reply:
Agreed, it would be nice to have